pandas/tests/arithmetic/test_period.py

```diff
@@ -1085,3 +1086,21 @@ def test_pi_sub_period_nat(self):
         exp = pd.TimedeltaIndex([np.nan, np.nan, np.nan, np.nan], name='idx')
         tm.assert_index_equal(idx - pd.Period('NaT', freq='M'), exp)
         tm.assert_index_equal(pd.Period('NaT', freq='M') - idx, exp)
+
+
+class TestPeriodArithmetic(object):
+
+    @pytest.mark.parametrize('freq,expected', [
+        (pd.offsets.Second, 18489600),
+        (pd.offsets.Minute, 308160),
+        (pd.offsets.Hour, 5136),
+        (pd.offsets.Day, 214),
+        (pd.offsets.MonthEnd, 7),
+        (pd.offsets.YearEnd, 1),
+    ])
+    def test_period_diff(self, freq, expected):
+        # GH 23878
+        for i in range(1, 4):
```

Comment:
you can `pytest.mark.parametrize` over this as well.

Reply:
@mroeschke : Perhaps it's lag, but you're repeating my comments from above.